Finished implementing MyStatsActivity

Gigantorz opened the pull request:
I am going to be working on the comments Activity on this branch as well

jJadsit reviewed (changes requested):
Doesn't crash but there is a couple errors with MyStatsActivity.
Firstly, there is an extra back button that doesn't need to be there.
Secondly, there are 3 categories we need to sort for: Points acquired, most scanned, and (which you don't have) highest scoring qr code found. Leaderboard controller should have a method for this too.
Finally, you need to use Leaderboard controller's findPlayer() method to set the text in this activity, right now you are setting the text to the array of players.

Gigantorz commented:
Ok In my recent commit I have taken care of the errors I think.
I have deleted the back button, that was a carry over from the first person who has made the activity. 
I have added another category, I apologize for missing it earlier, i didn't see it in the miro page outline.
And i didn't use findPlayer() to get the ranking, instead I have retrieved the arrayList for each ranking and then did .indexOf(p) for each of the ArrayList and displayed the index as its ranking.
So Far this is what I see 
![image](https://user-images.githubusercontent.com/69411983/161458066-31cad09c-86d5-40b1-9ab5-1dbb8906fd72.png)
Idk if its visible but, for each ranking on my Activity it is only showing the default "No Ranking" value since i don't have any scanned qr codes atm
